### ISSUE TITLE
ASpIRE dataset

### ISSUE THREAD

**Issue by gioannideImp** [NVIDIA/OpenSeq2Seq#471]
Hello, I am trying to transfer learn the Jasper model with the same vocabulary  trained on the LibriSpeech dataset (other+clean). It seems that the number of epochs is being calculated according to the number of global steps that have been completed. As a result, even though I trained the Jasper model for 14 epochs on Librispeech (that completed 123034 steps), when I run the ASpIRE dataset it displays epoch 1537. Additionally, it doesn't continue training and immediately finishes saying "Not enough steps for benchmarking". 

My question is: What should I do so that I can continue training with the ASpIRE dataset?

**Comment by blisc** [comment]
In OpenSeq2Seq, you can specify either `num_epochs` or `max_steps`.
You are correct that if you continue_learning, then the epoch calculation would be off, but I would advise you to try `max_steps` instead; and to please tune your learning rate scheduler.

**Comment by gioannideImp** [comment]
Thank you for the advice @blisc. What would you recommend for the learning rate scheduler?

**Comment by blisc** [comment]
Since learning rate scheduler is calculated based on current step number, I would just plot the learning rate on tensorboard to make sure that it is what you expect it to be.

IE most learning rates decay to some min number, just naively continuing without changing the steps in the scheduler will cause your job to train at this min lr instead of decaying or changing like you want it to
